daemon/algod/api/server/v2/generated/private/routes.go

```diff
@@ -39,9 +39,7 @@ type ServerInterfaceWrapper struct {
 // AbortCatchup converts echo context to params.
 func (w *ServerInterfaceWrapper) AbortCatchup(ctx echo.Context) error {
 
-	validQueryParams := map[string]bool{
-		"pretty": true,
-	}
+	validQueryParams := map[string]bool{}
```

Comment:
could you please check why we "lost" the `pretty` option ?

Reply:
resolved!